variables.pkr.hcl

```diff
@@ -24,93 +38,133 @@ variable client_secret {
+variable build_allowed_inbound_ip_addresses {
+  description = "List of IP addresses and CIDR blocks that should be allowed access to the VM during the building"
   type        = list(string)
 }
 
-variable replication_regions {
-  description = "A list of regions to replicate the image version in"
-  type        = list(string)
+// Note: Architecture specific
+variable build_vm_size {
+  description = "Size of the VM used for building"
+  type        = string
+  default     = "Standard_B1ls"
 }
 
-variable os_type {
-  description = "Type of the OS, this configures an SSH authorized key"
+variable build_os_type {
+  description = "Type of the OS, this configures the SSH authorized key"
   type        = string
   default     = "Linux"
 }
 
-variable image_offer {
+variable build_os_disk_size_gb {
+  description = "Size of the OS disk in GB (gigabytes). Values of zero or less than zero are ignored."
+  type        = number
+  default     = 30
+}
+
+#
+# Base image variables
+#
+
+// Note: Architecture specific
+variable base_image_offer {
   description = "Name of the publisher's offer to use for your base image"
   type        = string
   default     = "0001-com-ubuntu-server-jammy"
 }
 
-variable image_publisher {
+variable base_image_publisher {
   description = "Name of the publisher to use for your base image (Azure Marketplace Images only)"
   type        = string
   default     = "canonical"
 }
 
-variable image_sku {
+// Note: Architecture specific
+variable base_image_sku {
   description = "SKU of the image offer to use for your base image (Azure Marketplace Images only)."
   type        = string
   default     = "22_04-lts-gen2"
 }
 
-variable vm_size {
-  description = "Size of the VM used for building"
+#
+# Shared Image Gallery (SIG) variables
+#
+
+variable gallery_subscription_id {
+  description = "ID of the subscription where the Shared Image Gallery is located. Will use subscription_id as default if unspecified."
   type        = string
-  default     = "Standard_B1ls"
+  default     = null
+}
+
+variable gallery_resource_group {
+  description = "Name of the resource group where the Shared Image Gallery is located"
+  type        = string
+}
+
+variable gallery_name {
+  description = "Name of the Shared Image Gallery"
+  type        = string
+}
+
+// Note: Architecture specific
+variable gallery_image_definition {
+  description = "Name of the gallery definition for the built images of particular GraphDB version"
+  type        = string
+}
+
+variable gallery_image_replication_regions {
+  description = "A list of regions to replicate the image version in"
+  type        = list(string)
+  default     = []
 }
 
-variable image_replica_count {
+variable gallery_image_replica_count {
   description = "The number of replicas of the Image Version to be created per region. Replica count must be between 1 and 100"
   type        = number
   default     = 1
```

Comment:
better change this to  3

Reply:
I override this in the release pipeline as needed.